lib/Elastica/Bulk/ResponseSet.php

```diff
@@ -42,16 +42,13 @@ public function getBulkResponses()
      */
     public function getError()
     {
-        $error = '';
-
         foreach ($this->getBulkResponses() as $bulkResponse) {
             if ($bulkResponse->hasError()) {
-                $error = $bulkResponse->getError();
-                break;
+                return $bulkResponse->getError();
```

Comment:
@ReenExe I had a second look at our existing code and your changes. I made the decision to move away from the 1 return principle in Elastica as quite a few of your changes also make the code more readable. I will review your code a second time with this in mind.

Thanks for bringing up this topic and discussing it.